src/test/java/stepDefinitions/SEDTweaksSteps.java

```diff
@@ -1,107 +1,107 @@
+	{
+		driver = hooks.driver;
+		
+	}
+
+ 	
+	@Given("^I am on the Details page of Listing \"([^\"]*)\"$")
+	public void i_am_on_a_listing_s_details_pages(String arg1) throws Throwable {
+		driver.get("https://chpl.ahrqstg.org/#/product/" + arg1);
+		WebDriverWait wait = new WebDriverWait(driver, 20);
+		listingDetails_page.load_listings_details_page(driver); 	    
+	}
+
+	
+	@Then("^I should see the accordions in the following order: Criteria, CQMs, SED, G1/G2, Surveillance, Additional Info$")
+	public void i_should_see_the_accordions_in_the_following_order_Criteria_CQMs_SED_G1_G2_Surveillance_Additional_Info() throws Throwable {
+		
+		WebDriverWait wait = new WebDriverWait(driver, 20);
+		wait.until(ExpectedConditions.presenceOfElementLocated(By.id("details-certification-criteria")));
+		assertTrue(driver.findElement(By.id("details-certification-criteria")).getText().contains("Certification Criteria"));
+		assertTrue(driver.findElement(By.id("details-cqm")).getText().contains("Clinical Quality Measures"));
+		assertTrue(driver.findElement(By.id("details-sed")).getText().contains("Safety Enhanced Design (SED)"));
+		assertTrue(driver.findElement(By.id("details-g1g2")).getText().contains("Successfully Tested G1/G2 Measures"));
+		assertTrue(driver.findElement(By.id("details-surveillance-activities")).getText().contains("Surveillance Activities"));
+		assertTrue(driver.findElement(By.id("details-additional-information")).getText().contains("Additional Information"));
+		
+	}
+
+	@When("^I open SED details accordion$")
+	public void i_open_SED_accordion() throws Throwable {
+		listingDetails_page.sed_details_accordion_open(driver);
+	}
+	
+	@Then("^usability report text should read as 'Full Usability Report'$")
+	public void usability_report_title_reads_as_Full_Usability_Report() throws Throwable {
+		
+	assertTrue(driver.getPageSource().contains("Full Usability Report"));
+	}
+
+	@Then("^intended user description header should show as Description of Intended Users$")
+	public void the_header_should_be_named_Description_of_Intended_Users() throws Throwable {
+	    
+	assertTrue(driver.getPageSource().contains("Description of Intended Users"));
+			
+	}
+
+	@Then("^the header name should read as 'Date SED Testing was Completed'$")
+	public void the_header_should_be_named_as_Date_SED_Testing_was_Completed() throws Throwable {
+		
+	assertTrue(driver.getPageSource().contains("Date SED Testing was Completed"));
+	}
+
+	@Then("^the download button title should read as 'Download SED Details'$")
+	public void the_download_button_title_should_read_as_Download_SED_Details() throws Throwable {
+		
+	assertTrue(driver.getPageSource().contains("Download SED Details"));
+	}
+	
+	@Then("^the criteria number should also include the criteria title$")
+	public void the_criteria_number_should_also_include_the_criteria_title() throws Throwable {
+		assertTrue(driver.getPageSource().contains("170.315 (a)(1): Computerized Provider Order Entry (CPOE) - Medications"));

```

Comment:
Similar to above. This should call some sort of method in the PageObject that returns an Array of criteria titles, and then assert that various criteria w/titles are in that array. And probably that some other ones aren't; both criteria the Listing doesn't attest to, and criteria _without_ the title. E.g., `assertTrue(criteriaTitles.not.contains("170.315 (a)(3)"))`; where `criteriaTitles` is defined above as something like `String [] criteriaTitles = listingDetails.getCriteriaTitles()"`